src/tests/test_utils/functional_test_utils/layer_tests_summary/conformance_helper_tools/find_models_for_subgraphs_dumper.py

```diff
@@ -107,8 +120,9 @@ def generate_model_list_file(input_str: str, re_exp_file_path: str, output_file_
     logger.info(f"[ ARGUMENTS ] --filename={args.filename}")
     logger.info(f"[ ARGUMENTS ] --latest_only={args.latest_only}")
```

Comment:
Could you please add `exclude_models` as an arg of script with default value to the created file?